dll/win32/shell32/CDefView.cpp

```diff
@@ -612,6 +612,31 @@ void CDefView::UpdateStatusbar()
     }
 }
 
+unsigned __stdcall CDefView::_UpdateStatusbarProc(void *args)
+{
+    CDefView* pView = static_cast<CDefView*>(args);
+    pView->UpdateStatusbarWorker(pView->m_hUpdateStatusbarThread);
```

Comment:
I'd expect an `AddRef()` before starting this thread, and a `Release()` as last thing in this function